teku/src/main/java/tech/pegasys/teku/cli/options/ExecutionLayerOptions.java

```diff
@@ -96,6 +97,19 @@ public class ExecutionLayerOptions {
   private int builderCircuitBreakerAllowedConsecutiveFaults =
       DEFAULT_BUILDER_CIRCUIT_BREAKER_ALLOWED_CONSECUTIVE_FAULTS;
 
+  @Option(
+      names = {"--validators-builder-bid-challenge-percentage"},
+      paramLabel = "<STRING>",
+      showDefaultValue = Visibility.ALWAYS,
+      description =
+          "Fallback to local payload, when builder bid is available but its value could be beat by local payload. "
```

Comment:
we can simply say "Fallback to local payload whenever its value beats the builder bid.